cfc-devenv-base/.devcontainer/devcontainer.json

```diff
@@ -58,7 +55,6 @@
       "extensions": [
         "mhutchie.git-graph",
         "eamodio.gitlens",
-        "ms-azuretools.vscode-docker",
```

Comment:
Why we removing this extension?

Reply:
https://github.com/devcontainers/features/blob/main/src/docker-in-docker/README.md#vs-code-extensions it's included in the feature

Reply:
Ok